src/app/view/components/modal/settingsButton.js

```diff
@@ -0,0 +1,13 @@
+import elementCreator from '../../../utils/elementCreator';
```

Comment:
If this is a setting button, it shuold not be inside a modal folder. You could create a general button component in components folder and use it in modal thuogh